build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1027,6 +1027,10 @@ message ExecuteResponse {
   // phase. The keys SHOULD be human readable so that a client can display them
   // to a user.
   map<string, LogFile> server_logs = 4;
+
+  // Similar to `server_logs`, an optional list of URLs the user may visit to
+  // obtain more information about the execution of the action.
+  repeated string additional_info_url = 5;
```

Comment:
Why repeated? Do you have a use case for this already?

Reply:
I can only think this is due to logs from the worker side and logs from the execution side?

Reply:
That is a very good question. I have to say I didn't give this a lot of thought. "Why not?" must have been my reasoning.

Right now it's just the Bazel Buildbarn Browser to which we link. That said, there is a fair chance the scheduler processes will at some point gain a status page with info on workers, etc. Maybe there is a use for emitting links to that as well?

That said, that's all speculating. If there's a preference for a non-repeated field (or the other extreme: a string keyed map, just like the server logs), I can change this. :-)

Reply:
The more generic version of the `additional_info_url` field would be a 
```
string message = 5;
```
with the expectation being that a server can put an arbitrary message that a client is expected to display to the user. I do prefer the additional_info_url field because it's simple, no surprises and allows a client to have more control over how it presents the information to the user.

Reply:
> Right now it's just the Bazel Buildbarn Browser to which we link. That said, there is a fair chance the scheduler processes will at some point gain a status page with info on workers, etc. Maybe there is a use for emitting links to that as well?

Wouldn't this require either a key or a description for each URL to make this usable? A list of URLs without any metadata doesn't sound user-friendly.

Reply:
Yeah, exactly. Let's switch this to a map then.

Reply:
> Right now it's just the Bazel Buildbarn Browser to which we link. That said, there is a fair chance the scheduler processes will at some point gain a status page with info on workers, etc. Maybe there is a use for emitting links to that as well?

In that case couldn't you still only post a single link that takes the user to a web interface and link to other things from there? One link seems simple and powerful enough to fulfill all of these use cases.